validation/internal/legacy/l2outputoracle-1.3.0.go

```diff
@@ -0,0 +1,1218 @@
+// Code generated - DO NOT EDIT.
+// This file is a generated binding and any manual changes will be lost.
+
+package legacy
```

Comment:
Maybe a bit of a heavyweight way of solving this. I copypasted the bindings from an older commit in the monorepo.

Reply:
This seems ok for now. Maybe we add a TODO comment that this can be deleted once chains in the registry are on MCP (op-contracts/v1.3.0)?

Reply:
[add TODO comment](https://github.com/ethereum-optimism/superchain-registry/pull/157/commits/9d37013ddb9bdacb03746ddf802939fb290a1557)

Reply:
We could also strip out a lot of this bindings file, and just keep what we need?

Reply:
[56eefb8](https://github.com/ethereum-optimism/superchain-registry/pull/157/commits/56eefb8ecdd0f98a024c01a767cf3066cd384c57)

Reply:
Thank you!